build/jobs/provision-deploy.yml

```diff
@@ -22,6 +22,12 @@ parameters:
 - name: schemaAutomaticUpdatesEnabled
   type: string
   default: 'tool'
+- name: sqlServerAdminPassword
+  type: string
+  default: 'test@123'
```

Comment:
Could we remove the default value for this?

Reply:
Ah missed, done!